packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -349,7 +483,10 @@ const runCalibrate = async () => {
 			mappings: formattedMap
 		},
 		extra: {
-			num_iterations: knobs.value.numIterations
+			num_iterations: state.numIterations,
+			method: state.method,
+			step_size: state.stepSize,
+			learning_rate: state.learningRate
```

Comment:
Depend on how the component is setup, which you may need to ask Tom. "state" can be one step behind "knobs" because knobs are the reactive part whereas state came from the parent, so generally speaking we want to send the request with the values embedded under knobs rather than state.

The rough semantic is as such:
- modeller tweak "knobs"
- modeller send request to do stuff
- "knobs" merged back into "state" to sync up parent/child